doc/data-warehouse-what-and-why.md

```diff
@@ -0,0 +1,258 @@
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Table of Contents**
+
+- [The need for a Data Warehouse](#the-need-for-a-data-warehouse)
+    - [Long term vision](#long-term-vision)
+    - [Give departments the data they need](#give-departments-the-data-they-need)
+    - [At scale and over time](#at-scale-and-over-time)
+        - [Example 1: Number of Content Items published by an organization](#example-1-number-of-content-items-published-by-an-organization)
+        - [Example 2: Enforcing good practices](#example-2-enforcing-good-practices)
+        - [Example 3: Predictions](#example-3-predictions)
+- [Why a Data Warehouse](#why-a-data-warehouse)
+    - [Overview](#overview)
+    - [Track content changes over time for Content Items](#track-content-changes-over-time-for-content-items)
+    - [A new application for GOV.UK?](#a-new-application-for-govuk)
+
+<!-- markdown-toc end -->
+
+# The need for a Data Warehouse
+
+This document is not an in-depth explanation of what is Data Warehouse but an
+explanation of the reasons behind building it. There are many online resources
+and books about Data Warehouses if you want to get further information about
+this matter.
+
+To understand the foundation for the Data Warehouse, let's start with the `long
+term vision` that guided our work.
+
+##  Long term vision
+
+```
+Give departments the data they need, at the point they need it so that they can
+manage their GOV.UK content at scale over time.
+```
+
+Let's explore in detail:
+
+## Give departments the data they need
+
+Departments manage content daily. They publish, update and archive content as
+part of their daily work; as of today, content designers don't have a single
```

Comment:
`content designers don't have a single place to get metrics about how their content is performing.` reads like content designers can't currently get any metrics. 

maybe something like -
`content designers have to search across multiple sources or tools to find metrics. Some of these sources are inaccessible, difficult to use or require further analysis. The aim of the Data Warehouse is to provide a single location for content designers to see all relevant metrics about their content.`